modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/GETTING_STARTED.markdown

```diff
@@ -149,6 +149,18 @@ $h4 liferay.workspace.bundle.dist.include.metadata
 Set this to true to append metadata for the current environment settings and
 timestamp. The default value is `false`.
 
+$h4 liferay.workspace.docker.access.token
+See this page (https://docs.docker.com/docker-hub/access-tokens) on how to generate a docker access token.
+
+$h4 liferay.workspace.docker.local.registry.address
+Set this to the host and port of the local docker registry. This will enable the user to interact with a docker registry other than dockerhub, provided the user has access permissions. Ex: myregistryaddress.org:5000
+
+$h4 liferay.workspace.docker.pull.policy
+Set this to true to pull the user's local docker cache first. The default value is true.
```

Comment:
I thought the default value right now is false?

Reply:
for now, in the code , the default value is true, should I change it to "false"?
https://github.com/liferay/liferay-portal/blob/29c7086d2a830e711dfa551844b2a77a62369d93/modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java#L119